cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowConditionNode.java

```diff
@@ -42,7 +59,25 @@ public WorkflowConditionNode(String nodeId, String predicateClassName, List<Work
     return elseBranch;
   }
 
+  @Nullable
```

Comment:
Why this can be null? It's changing API contract here. If the predicate is coming from a `Condition` class, the `Condition` class name is the predicate class name, isn't it?